src/app/ui/editor/pixels_movement.cpp

```diff
@@ -369,14 +392,27 @@ void PixelsMovement::moveImage(const gfx::PointF& pos, MoveModifier moveModifier
   gfx::RectF bounds = m_initialData.bounds();
   gfx::PointF abs_initial_pivot = m_initialData.pivot();
   gfx::PointF abs_pivot = m_currentData.pivot();
+  const bool tilesMode = m_site.tilemapMode() == TilemapMode::Tiles;
```

Comment:
```suggestion
  const bool tilesMode = (m_site.tilemapMode() == TilemapMode::Tiles);
```
Please use `(...)` for these cases.